integration/Substra/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Other
+local-worker/
+experiment_summaries/*
+!experiment_summaries/.gitkeep
```

Comment:
just to share another way of doing this: 

do not put anything in this `.gitignore`
in the `experiment_summaries` folder, add a `.gitignore` file with the following contents:

```
*
!.gitignore
```